lib/consul/async/utilities.rb

```diff
@@ -1,9 +1,16 @@
 require 'tempfile'
 require 'yaml'
 require 'json'
+
 module Consul
   module Async
     class Utilities
+      def self.valid_json?(json)
```

Comment:
This method might be expensive: it parses everything just to know whether the value is correct JSON while not doing anything with the result

Reply:
See comment on other json hazardeous parsing, will avoid parsing response multiple time